jupyterlab_bigquery/src/components/details_panel/details_panel.tsx

```diff
@@ -45,12 +45,24 @@ export const localStyles = stylesheet({
   },
 });
 
+interface ChipProps {
+  darkMode: boolean;
+  size?: 'medium' | 'small';
+  component?: any;
+  key?: string | number;
+  label: string;
+}
+
 const StyledChip = withStyles({
   root: {
-    color: '#1967D2',
-    backgroundColor: 'rgba(25, 103, 210, 0.1)',
+    color: (props: ChipProps) =>
+      // white :  blue600
+      props.darkMode ? 'var(--jp-ui-font-color1)' : '#1A73E8',
+    backgroundColor: (props: ChipProps) =>
+      // blue300 at 30% opacity : blue600 at 10% opacity
+      props.darkMode ? 'rgba(138, 180, 248, 0.3)' : 'rgba(26, 115, 232, 0.1)',
   },
-})(Chip);
+})((props: ChipProps) => <Chip {...props} />);
```

Comment:
For any objects that need to be styled with both the `withStyles` HOC and by reading the body attribute `data-jp-theme-light`, the body attribute or color has to be passed in to the style sheet through props like this.